docs/release-notes/release-notes-0.17.0.md

```diff
@@ -72,6 +72,9 @@ package](https://github.com/lightningnetwork/lnd/pull/7356)
   `grpc.keepalive_time_ms=5100` is recommended on the client side (adding 100ms
   to account for slightly different clock speeds).
 
+* [Fixed a bug where we didn't check for correct networks when submitting
+  onchain transactions](https://github.com/lightningnetwork/lnd/pull/6448).
+
 ## Misc
```

Comment:
`sendcoins` change seems to be missing from the release notes. Given that it's a breaking change (for shell script apps), we must include the change here.

Reply:
Good catch, added!